### PR TITLE
Minor docs improvements

### PR DESCRIPTION
**Pull request by eteeselink**
Hi there! Thanks for your great work. It took me some effort to understand a number of things, so I decided I might as well help out and clarify the things I did not understand. So here's a small documentation PR, about custom builds and webpack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +124,8 @@ require('core-js/build')({
   // ...
 });
 ```
+If your webpack build includes something like UglifyJs, then the excluded parts will be entirely stripped from the resulting code using dead code elimination.
```
Comment (zloirock) [inline README.md]: The excluded parts just will not be added to the bundle :)

Comment (eteeselink) [inline README.md]: Ahyes, i see why that would be confusing. We dont want webpack removing our input code :-) Will update.

File: README.md [hunk 2]
```diff
@@ -70,7 +70,11 @@ var core = require('core-js/library');
 // Shim only
 require('core-js/shim');
 ```
-If you need complete build for browser, use builds from `core-js/client` path:  [default](https://raw.githack.com/zloirock/core-js/v1.2.6/client/core.min.js), [without global namespace pollution](https://raw.githack.com/zloirock/core-js/v1.2.6/client/library.min.js), [shim only](https://raw.githack.com/zloirock/core-js/v1.2.6/client/shim.min.js).
+If you need complete build for browser, use builds from `core-js/client` path:  
+
+* [default](https://raw.githack.com/zloirock/core-js/v1.2.6/client/core.min.js): Includes all features, standard and non-standard.
+* [as a library](https://raw.githack.com/zloirock/core-js/v1.2.6/client/library.min.js): Like "default", but does not pollute the global namespace (see example below).
```
Comment (zloirock) [inline README.md]: For this bundle - rather the example at the top of the readme.

File: README.md [hunk 3]
```diff
@@ -70,7 +70,11 @@ var core = require('core-js/library');
...
+* [shim only](https://raw.githack.com/zloirock/core-js/v1.2.6/client/shim.min.js): Only includes the standard ECMAScript methods.
```
Comment (zloirock) [inline README.md]: Not only ECMAScript, also web standards / proposals like `setImmediate`.

Comment (eteeselink) [inline README.md]: Good one. I'll just do "Only includes standard methods". Good?

Comment (zloirock) [inline README.md]: Sure.

File: README.md (hunk 1 above)
Comment (zloirock) [inline README.md]: The excluded parts will be missing *before* uglifyjs, so you can just remove this paragraph.

**Comment by zloirock** [comment]
Please, fix the last problem, rebase it as one commit and I'll merge it.

**Comment by eteeselink** [comment]
done! thanks for holding my hand a bit, i don't contribute to OSS that much yet.

File: README.md (hunk 1 above)
Comment (eteeselink) [inline README.md]: Sorry, missed this comment earlier. Are you entirely sure about this? I'm pretty certain that if you do

    if(false) { 
        require('something'); 
    }

then webpack just nicely expands the require and leaves it up to a cleanup tool to remove the dead code. (https://webpack.github.io/docs/list-of-hints.html - "Use the DefinePlugin to pass configuration from config to app: Embedded into bundle, Conditional code is removed with minimized").

Or am I just completely misunderstanding how core-js/build works? :-)

**Comment by eteeselink** [comment]
Sorry, i missed one comment, see above. Will rebase again once it's all done.

File: README.md (hunk 1 above)
Comment (zloirock) [inline README.md]: If you will exclude `something` - `require('something')` just will not be added :)

**Comment by eteeselink** [comment]
All done!

**Comment by zloirock** [comment]
Thanks.

**Comment by eteeselink** [comment]
Likewise!

**Pull request merged**